schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -465,7 +473,9 @@ def read_scheduler(self):
             return False
 
     def make_scheduler_summary_df(self):
-        """Update conditions, make the reward and scheduler summary dataframes.
+        """
```

Comment:
The summary line for a docstring should be on the same line as the initial tripple quote (see https://developer.lsst.io/python/numpydoc.html , in the section whose heading starts with "Docstrings SHOULD begin with".